c/httpserver.c

```diff
@@ -83,6 +83,17 @@ typedef int time_t;
 #define APF_AUTHORIZED  1
 #endif
 
+#define SAF_RC_PASSWORD_EXPIRED    8
+#define RACF_RC_PASSWORD_EXPIRED   12
+
+typedef struct AuthResponse_tag {
```

Comment:
This struct and defines are going to see a lot of use in the future I think, so this should be in a .h file rather than embedded in this .c file.
I'm not sure which .h file is best: is it one related to the httpserver, or one related to auth

Reply:
irek said to leave it in the .c file since its only used here

Reply:
@1000TurquoisePogs I think `AuthResponse` is very specific to `httpserver.c`. We should move it when we need to reuse it, right now there is only one user.